modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -8,39 +8,179 @@
 
 Configure disk partitioning for a managed cluster using a `SiteConfig` CR and {ztp-first}. The  disk partition details in the `SiteConfig` CR must match the underlying disk.
 
-[NOTE]
+[IMPORTANT]
 ====
-Use persistent naming for devices to avoid device names such as `/dev/sda` and `/dev/sdb` being switched at every reboot. You can use `rootDeviceHints` to choose the bootable device and then use same device for further partitioning.
+You must complete this procedure at installation time.
 ====
 
 .Prerequisites
 
-* You have installed the OpenShift CLI (`oc`).
+* Install Butane.
 
-* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+.Procedure
 
-* You have created a Git repository where you manage your custom site configuration data for use with {ztp-first}.
+. Create the `storage.bu` file.
++
+[source,yaml]
+----
+variant: fcos
+version: 1.3.0
+storage:
+  disks:
+  - device: /dev/disk/by-path/pci-0000:01:00.0-scsi-0:2:0:0 <1>
+    wipe_table: false
+    partitions:
+    - label: var-lib-containers
+      start_mib: <start_of_partition> <2>
+      size_mib: <partition_size> <3>
+  filesystems:
+    - path: /var/lib/containers
+      device: /dev/disk/by-partlabel/var-lib-containers
+      format: xfs
+      wipe_filesystem: true
+      with_mount_unit: true
+      mount_options:
+        - defaults
+        - prjquota
+----
+<1> Specify the root disk.
+<2> Specify the start of the partition in MiB. If the value is too small, the installation will fail.
+<3> Specify the size of the partition. If the value is too small, the deployments after installation will fail.
 
-.Procedure
+. Convert the `storage.bu` to an Ignition file.
++
+--
+[source,terminal]
+----
+$ butane storage.bu
+----
 
-. Add the following YAML that describes the host disk partitioning to the `SiteConfig` CR that you use to install the managed cluster:
+.Example output
+[source,terminal]
+----
+{"ignition":{"version":"3.2.0"},"storage":{"disks":[{"device":"/dev/disk/by-path/pci-0000:01:00.0-scsi-0:2:0:0","partitions":[{"label":"var-lib-containers","sizeMiB":0,"startMiB":250000}],"wipeTable":false}],"filesystems":[{"device":"/dev/disk/by-partlabel/var-lib-containers","format":"xfs","mountOptions":["defaults","prjquota"],"path":"/var/lib/containers","wipeFilesystem":true}]},"systemd":{"units":[{"contents":"# # Generated by Butane\n[Unit]\nRequires=systemd-fsck@dev-disk-by\\x2dpartlabel-var\\x2dlib\\x2dcontainers.service\nAfter=systemd-fsck@dev-disk-by\\x2dpartlabel-var\\x2dlib\\x2dcontainers.service\n\n[Mount]\nWhere=/var/lib/containers\nWhat=/dev/disk/by-partlabel/var-lib-containers\nType=xfs\nOptions=defaults,prjquota\n\n[Install]\nRequiredBy=local-fs.target","enabled":true,"name":"var-lib-containers.mount"}]}}
+----
+--
+
+. Use a tool like link:https://jsonformatter.org/json-pretty-print[JSON Pretty Print] to convert the output into JSON format. 
```

Comment:
As per IBM SG. Do not use "like" to mean "such as". Rewrite sentence: "Use a tool such as..."